qemu/tests/cfg/migration_with_block.cfg

```diff
@@ -53,25 +53,24 @@
                     cdrom_orig_name = orig
                     cdroms = ${cdrom_orig_name}
                     image_boot_orig = yes
-                    cdrom_orig = /tmp/${cdrom_orig_name}.iso
+                    cdrom_orig = /var/tmp/${cdrom_orig_name}.iso
                     cdrom_orig_file = ${cdrom_orig}
                     cdrom_new_name = new
-                    cdrom_new_file = /tmp/${cdrom_new_name}.iso
+                    cdrom_new_file = /var/tmp/${cdrom_new_name}.iso
                     pre_command_noncritical = no
-                    pre_command = 'dd if=/dev/zero of=/tmp/${cdrom_orig_name} bs=1M count=1024 && '
-                    pre_command += 'mkisofs -o /tmp/${cdrom_orig_name}.iso /tmp/${cdrom_orig_name} && '
-                    pre_command += 'dd if=/dev/zero of=/tmp/${cdrom_new_name} bs=1M count=2048 && '
-                    pre_command += 'mkisofs -o /tmp/${cdrom_new_name}.iso /tmp/${cdrom_new_name}'
-                    post_command = "rm -rf /tmp/${cdrom_new_name}.iso /tmp/${cdrom_new_name} "
-                    post_command += "/tmp/${cdrom_orig_name}.iso /tmp/${cdrom_orig_name}"
-                    check_items = '{"io-status": "ok", "device": "drive_${cdrom_orig_name}", "tray_open": False'
-                    check_orig_items =  '${check_items}, "file": "${cdrom_orig_file}"}'
-                    check_new_items =  '${check_items}, "file": "${cdrom_new_file}"}'
+                    pre_command = 'dd if=/dev/zero of=/var/tmp/${cdrom_orig_name} bs=1M count=1024 && '
+                    pre_command += 'mkisofs -o /var/tmp/${cdrom_orig_name}.iso /var/tmp/${cdrom_orig_name} && '
+                    pre_command += 'dd if=/dev/zero of=/var/tmp/${cdrom_new_name} bs=1M count=2048 && '
+                    pre_command += 'mkisofs -o /var/tmp/${cdrom_new_name}.iso /var/tmp/${cdrom_new_name}'
+                    post_command = "rm -rf /var/tmp/${cdrom_new_name}.iso /var/tmp/${cdrom_new_name} "
+                    post_command += "/var/tmp/${cdrom_orig_name}.iso /var/tmp/${cdrom_orig_name}"
+                    check_orig_items = '{"io-status": "ok", "device": "drive_${cdrom_orig_name}", "tray_open": False, "file": "${cdrom_orig_file}"}'
+                    check_new_items = '{"io-status": "ok", "device": "%s", "tray_open": False, "file": "${cdrom_new_file}"}'
```

Comment:
Refer to https://github.com/autotest/tp-qemu/pull/1739 reviewing test results, It works. In other words, this value of `node-name` from the output of qmp command `query-block` doesn't change after cdrom changed. but now it doesn't work. So this behavior of qemu is expected? I mean this `node-name` will be the same in some versions and will be changed to a random value in other versions after cdrom changed. 

From my view, I think the check items should not be given by routine dynamically since those are we expect value but up to routine.

Reply:
Hi @YongxueHong 

I also have the same concern before, I confirmed with qinwang, you can check this Bug 1922964 - [throttle] lost throttle attributes on cdrom device after execute blockdev-change-medium operation. 
As the developer reply: The observed behaviour is actually correct, **blockdev-change-medium is a shortcut for creating a format and protocol node like '-drive filename=...' would create and then replacing the whole attached block tree for the device.** The old tree contained a 'throttle' node, but the newly created one doesn't, so losing the throttling functionality is expected.